server/models/Card.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require("mongoose");
+
+const cardSchema = new mongoose.Schema({
+  _id: {
+    type: mongoose.Types.ObjectId,
+    required: true,
+  },
+  name: {
+    type: Array,
+    required: false,
+  },
+  tag: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: false,
+  },
+  deadline: {
+    type: Date,
+    required: false,
+  },
+  comment: {
+    type: String,
+    required: false,
+  },
+  attachment: {
+    type: String,
+    required: false,
+  },
+  checklist: {
+    type: Array,
+    required: false,
+  },
+  columnId: {
+    type: mongoose.Types.ObjectId,
+    required: true,
+  },
+  index: {
+    type: Number,
+    required: true,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now,
+  },
+});
```

Comment:
Mongoose supports timestamps by setting the second parameter in the Schema contstructor params `{ timestamps: true }`. They'll even support updatedAt too